gradle.properties

```diff
@@ -1,7 +1,8 @@
 kotlin.code.style=official
 
 #Gradle
+#Memory bumped to 4 GiB because otherwise we were getting "GC overhead limit exceeded".
```

Comment:
Revert changes in this file because wasm tests are still flaky, and disable wasm tests instead.